cub/cub/device/dispatch/dispatch_histogram.cuh

```diff
@@ -1586,6 +1592,7 @@ public:
     return error;
   }
 
+#ifndef DOXYGEN_SHOULD_SKIP_THIS // Do not document
   CUB_RUNTIME_FUNCTION _CCCL_FORCEINLINE static cudaError_t DispatchEven(
```

Comment:
Unrelated: this overload is missing a `CUB_DETAIL_RUNTIME_DEBUG_SYNC_IS_NOT_SUPPORTED` annotation.